tests/dataset_processor_test.py

```diff
@@ -188,6 +180,88 @@ def test_dataset_processor_t5_style():
     gc.collect()
 
 
+def test_dataset_processor_non_numerical_column():
+    """Test the `process_dataset_dict` function of T5-type `TextualizeProcessor`."""
```

Comment:
We should test a numerical column, right? 🤔

Reply:
Oh right 🤦  fixed the comment